csrankings-j.csv

```diff
@@ -1695,6 +1695,7 @@ Jonathan Gaudreault,Université Laval,https://www.ift.ulaval.ca/departement-et-p
 Jonathan Gemmell,DePaul University,http://www.cdm.depaul.edu/about/Pages/People/facultyinfo.aspx?fid=494,RIhZrvIAAAAJ
 Jonathan I. Maletic,Kent State University,http://www.cs.kent.edu/~jmaletic,n9_W4kYAAAAJ
 Jonathan K. Lazar,University of Maryland - College Park,https://ischool.umd.edu/faculty-staff/jonathan-lazar,CDpfbBsAAAAJ
+Jonathan Kelly,University of Toronto,https://starslab.ca/,KtSR8_0AAAAJ

```

Comment:
Not in CS.

Reply:
You have commented elsewhere that Cross-Appointment qualifies. 
https://github.com/emeryberger/CSrankings/pull/2629#issuecomment-621956183

these are cross-appointed faculty in CS. 
https://web.cs.toronto.edu/people/faculty-directory#crossappointed

Other cases of Cross-appointed faculty in CS
- Alex Mihailidis
- Brendon Frey
- Natalie Jerger: https://github.com/emeryberger/CSrankings/pull/2117

Including some but not others would not be fair.

Reply:
Thanks. I think since Toronto has a more porous system of affiliations, faculty don't always list all of their affiliations. 
But I hope the formal department page with cross-appointments will suffice as evidence.